man/useradd.8.xml

```diff
@@ -464,6 +464,10 @@
 	    <option>-m</option> options if you want a home directory for a
 	    system account to be created.
 	  </para>
+	  <para>
+	    Note that <command>useradd</command> with this option will not
+	    update <filename>/etc/subuid</filename> and <filename>/etc/subgid</filename>.
```

Comment:
Check that the number of characters in a line is below 80.

Reply:
Updated.